chrome/browser/net/chrome_network_delegate.cc

```diff
@@ -62,6 +62,7 @@
 #if defined(OS_ANDROID)
 #include "base/android/path_utils.h"
 #include "chrome/browser/io_thread.h"
+#include <regex>
```

Comment:
Since the code isn't ANDROID specific, this include shouldn't be in that `#if...#endif` block. Moving it to around line 11 seems to be the best place.